deploy/chart/templates/rbac.yaml

```diff
@@ -228,6 +228,10 @@ rules:
   - apiGroups: [""]
     resources: ["nodes/stats"]
     verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    resourceNames: ["ossfs-mime-config"]
+    verbs: ["get"]
```

Comment:
Will this be called from node or controller? I think it should be controller, because the fuse pod is created there, right? If it is really called from node, it can be blocked by node-restriction if enabled, despite you adding RBAC permission for it.